x-pack/test/functional/apps/dashboard/group1/reporting/screenshots.ts

```diff
@@ -47,7 +47,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
           indices: [
             {
               names: ['ecommerce', 'kibana_sample_data_ecommerce'],
-              privileges: ['read'],
+              privileges: ['read', 'view_index_metadata'],
```

Comment:
Let's not change permissions here as this is a valid test case that caught a pretty big error.

Reply:
Aye aye, will undo this now that the fix is merged in!